fury/window.py

```diff
@@ -32,6 +33,24 @@ class Scene(Renderer):
     but also it provides access to all the functionality
     available in ``vtkRenderer`` if necessary.
     """
+    def __init__(self, background=(0, 0, 0), skybox_tex=None,
+                 render_skybox=False):
+        self.skybox = skybox_tex
+        self.render_skybox = render_skybox
+        self.skybox_actor = None
+        if skybox_tex:
+            self.AutomaticLightCreationOff()
+            self.UseImageBasedLightingOn()
+            self.UseSphericalHarmonicsOff()
```

Comment:
if you use the `if`, this line does not work with vtk 9.0.3  So I confirm that we should use vtk 9.1.0 as minimum version